airgun/entities/container.py

```diff
@@ -19,11 +19,44 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def read(self, entity_name, widget_names=None):
+        """Read container details."""
+        view = self.navigate_to(self, 'Details', entity_name=entity_name)
+        return view.read(widget_names=widget_names)
+
+    def set_power(self, entity_name, power):
+        """Power on or power off specific container.
+
+        Args:
+            entity_name: name of container
+            power: power state to set container into. May be one of "On"/"Off"/True/False.
+        """
+        view = self.navigate_to(self, 'Details', entity_name=entity_name)
+        if power in (True, 'On'):
+            view.power_on.click()
+        elif power in (False, 'Off'):
+            view.power_off.click(handle_alert=True)
+        else:
+            raise ValueError(
+                'Wrong `power` value passed. Supported values are True/False/"On"/"Off"')
+        view.flash.assert_no_error(
+            ignore_messages=['Error - wrong number of arguments (given 1, expected 0)'])
```

Comment:
what is that one about?

Reply:
[BZ1683348](https://bugzilla.redhat.com/show_bug.cgi?id=1683348) just some weird message appears, but container stops/starts successfully

Reply:
I think comment with BZ id will not harm much